sites/gc-page-feedback/gc-page-feedback-custom-en.html

```diff
@@ -0,0 +1,32 @@
+---
+{
+	"altLangPage": "gc-page-feedback-custom-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"css": ["https://use.fontawesome.com/releases/v5.15.4/css/all.css"],
+	"dateModified": "2022-09-21",
+	"description": "Feedback form for Canada.ca",
+	"language": "en",
+	"title": "Page feedback form with custom parameters",
+	"pageFeedback": {
+		"institutionopt": "Custom institution",
+		"themeopt": "Custom theme",
+		"sectionopt": "Custom section",
+		"pageTitle": "true",
+		"language": "true",
+		"submissionPage": "true"
+	}
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<h2>HTML code</h2>
+{% highlight text %}
+<footer class="pagedetails">
+<h2 class="wb-inv">Page details</h2>
+<div class="row">
+{% include_relative includes/gc-page-feedback.html %}<div class="wb-share col-sm-4 col-md-3 col-sm-offset-1 col-md-offset-2 col-lg-offset-3" data-wb-share='{"lnkClass": "btn btn-default btn-block"}'></div>
```

Comment:
The purpose of the "custom" demo pages is (was?) to demonstrate how to override the widget's form parameters via the ``pageFeedback`` Jekyll variable ([lines 12-19](https://github.com/wet-boew/GCWeb/blob/e3b65b7bf18ebb20e21400551e1542209c45c342/sites/gc-page-feedback/gc-page-feedback-custom-en.html#L12-L19)). That allowed page-specific parameters to be "baked" into the page's hardcoded HTML - which allowed the widget to pass them over in basic HTML + noscript scenarios.

I'm open to removing the custom demo pages if needed. But I just wanted to make everyone understands their "raison d'être" and the implications of removing them beforehand.

Reply:
@EricDunsworth I do agree, did you get @duboisp's opinion on this as well?

Reply:
I haven't spoken with @duboisp about this.

@GormFrank I think I'll keep the custom demo pages in place until your team takes over this PR. They could potentially come in handy if your side decides to officially-support custom OPUFT parameters in GCWeb's Jekyll theme (since I've already done all the legwork in ``sites/gc-page-feedback/includes/parameters.liquid`` and was just using the custom demo pages to show it off).